src/lib.rs

```diff
@@ -91,21 +91,38 @@ impl From<u32> for Pid {
 }
 
 
-#[cfg(feature = "log")]
+#[cfg(feature = "tracing")]
 #[macro_use]
-mod log {
+mod tracing {
     #[allow(unused)]
-    pub(crate) use log::debug;
-    pub(crate) use log::error;
+    pub(crate) use tracing::debug;
+    pub(crate) use tracing::error;
     #[allow(unused)]
-    pub(crate) use log::info;
+    pub(crate) use tracing::info;
     #[allow(unused)]
-    pub(crate) use log::trace;
-    pub(crate) use log::warn;
+    pub(crate) use tracing::trace;
+    pub(crate) use tracing::warn;
+
+    #[allow(unused)]
+    pub(crate) use tracing::debug_span;
+
+    #[allow(unused)]
+    pub(crate) use tracing::error_span;
+
+    #[allow(unused)]
+    pub(crate) use tracing::info_span;
+
+    #[allow(unused)]
+    pub(crate) use tracing::trace_span;
+
+    #[allow(unused)]
+    pub(crate) use tracing::warn_span;
```

Comment:
> Let's add those when we need them, perhaps?

that would work too i added them because i saw `#[allow(unused)]` for `log::info` and others so i thought we could have these as well for later

Reply:
It's subjective, but the others (`info!`, ...) are "core" primitives where once we use one we probably want the others eventually. I don't believe I have ever actually used the `*_span` functionality, to be honest, so it may make more sense to add once we do. Once that is the case, though, it's probably fine to add all the variants (`info`, `error`, etc).

Reply:
> It's subjective, but the others (`info!`, ...) are "core" primitives where once we use one we probably want the others eventually. I don't believe I have ever actually used the `*_span` functionality, to be honest, so it may make more sense to add once we do. Once that is the case, though, it's probably fine to add all the variants (`info_span`, `error_span`, etc).

cool makes sense